CMakeLists.txt

```diff
@@ -46,6 +77,7 @@ if(WIN32 AND MSVC_VERSION LESS 1900)
   message(FATAL_ERROR "Building with Microsoft compiler needs Latest Visual Studio 2015 (Community or better)")
 endif()
 
+option(ENABLE_MASON "Use mason for dependencies" OFF)
```

Comment:
`ENABLE_MASON` conflicts with `ENABLE_LTO` and `ENABLE_GOLD_LINKER` so there should be a warning and force-disable for both.